src/framework/preferences/html/PrefsEditorTemplate-speak.html

```diff
@@ -1,11 +1,10 @@
 <h2><span class="fl-icon-speak"></span><span class="flc-prefsEditor-speak-label heading-text"></span></h2>
-
 <div class="fl-prefsEditor-onoff">
     <input type="checkbox" id="speak-choice" class="flc-prefsEditor-speak" />
     <label for="speak-choice">
+    	<p class="flc-prefsEditor-speak-choice-label"></p>
```

Comment:
I left a comment about this below. Also if you use html tags straight in the comments here, they will be treated as actual markup, because these comment fields support markdown. You should wrap the tag name in ` to have it styled as a code. You can look at the [GitHub Flavoured Markdown](https://help.github.com/articles/github-flavored-markdown/) page for more info on syntax.